app/javascript/mastodon/components/avatar_overlay.tsx

```diff
@@ -18,27 +18,33 @@ export const AvatarOverlay: React.FC<Props> = ({
   baseSize = 36,
   overlaySize = 24,
 }) => {
-  const { hovering, handleMouseEnter, handleMouseLeave } = useHovering(autoPlayGif);
-  const accountSrc = hovering ? account?.get('avatar') : account?.get('avatar_static');
-  const friendSrc = hovering ? friend?.get('avatar') : friend?.get('avatar_static');
+  const { hovering, handleMouseEnter, handleMouseLeave } =
+    useHovering(autoPlayGif);
```

Comment:
Is that way of having the right-hand side entirely on a new line really more readable?

Reply:
The whole line is > 80 char, so Prettier reformats it to be under this limit by moving the right-hand side to the new line